### DISCUSSION TITLE
Alist添加HTTPS配置后，套CF访问会521错误

### DISCUSSION THREAD

**Discussion by Niomaor** [AlistGo/alist#7339] [Q&A]
在配置文件文件中，`site_url`已设置为`https://xxx.example.com`了
在`scheme`也已经添加了HTTPS端口以及CloudFlare证书
CloudFlare也已经开启代理
但问题是不管怎么搞，网页始终打开只有Error code 521
求大佬指点

**Comment by sd14** [comment]
1，SSL/TLS设置里如果是灵活，要改，或者去规则Configuration Rules里加一条最下面那里改模式
2，HTTPS用什么端口，如果不是cf默认支持的端口，去规则Origin Rules里加一条改端口
